RecoTracker/LSTCore/interface/MiniDoubletsSoA.h

```diff
@@ -0,0 +1,61 @@
+#ifndef RecoTracker_LSTCore_interface_MiniDoubletsSoA_h
+#define RecoTracker_LSTCore_interface_MiniDoubletsSoA_h
+
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+#include "DataFormats/Portable/interface/alpaka/PortableCollection.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
```

Comment:
uhm, I wouldn't expect `ALPAKA_ACCELERATOR_NAMESPACE` to show up in a simple `interface/` subdir.
Only `*Device*Collection` would be declared first in this namespace

Reply:
Oh sorry, yeah I need to fix that

Reply:
I think there's three options:
1. Move the header to `interface/alpaka`.
2. Keep it in `interface` and define the DeviceCollection in some alpaka header.
3. Keep it in `interface` and define the DeviceCollection with a template.

I'm leaning towards 2.

Reply:
2. looks better long term